cirq-google/cirq_google/engine/engine.py

```diff
@@ -59,6 +59,11 @@
 _R = TypeVar('_R')
 
 
+# Feature gate for making Quantum Engine requests using the stream RPC.
+# TODO(#5996) Remove the flag once the feature is stable.
+_STREAM_FEATURE_FLAG = True
```

Comment:
As part of moving this flag to `EngineContext`, I renamed it to `enable_streaming`, since technically the stream contains both requests and responses. Open to name suggestions, though!